includes/admin/class-wcmypa-admin.php

```diff
@@ -996,7 +999,8 @@ private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
         }
 
         $confirmationData = [
-            __("delivery_type", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
+            __("delivery_type", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman(
+            )[$deliveryOptions->getDeliveryType()],
```

Comment:
Ik zou deze gewoon op 1 regel houden, ik denk dat het dan overzichtelijker is

Reply:
done